Makefile

```diff
@@ -0,0 +1,45 @@
+# Variables
+ENV_NAME = credit_card_defaults_prediction            # Name of the conda environment
+PYTHON_VERSION = 3.10.15         # Default Python version if environment.yml is not present
+ENV_FILE = environment.yml   # Name of the environment file
+
+# Targets
+
+.PHONY: create update conda-env clean activate deactivate notebook freeze verify
+
+create: conda-env
+
+# Create a conda environment from environment.yml
+conda-env:
+	@echo "Creating conda environment $(ENV_NAME) from $(ENV_FILE)..."
+	conda create  --name $(ENV_NAME) --file $(ENV_FILE)
```

Comment:
This is missing the python version

Reply:
Ok. Now I saw the python version in dependencies as well. In that case, it could be like this:
```
if [ -f "$(ENV_FILE)" ]; then \
	  conda env create --name $(ENV_NAME) --file $(ENV_FILE); \
	else \
	  conda create --name $(ENV_NAME) python=$(PYTHON_VERSION); \
	fi
```

Reply:
Honestly, I think we can just assume the env file will always be there, in which case, remove the definition of
```PYTHON_VERSION = 3.10.15```